### PR TITLE
[Doc] Update docstring for module misc

### PR DESCRIPTION
**Pull request by neozhaoliang**

### CODE REVIEW

File: python/taichi/lang/impl.py [hunk 1]
```diff
@@ -367,13 +367,17 @@ def reset():
 
 @taichi_scope
 def static_print(*args, __p=print, **kwargs):
+    """The print function in Taichi scope. This function is called
```
Comment (k-ye) [inline python/taichi/lang/impl.py:370]: Note that docstring usually starts with a one line short-line description

File: python/taichi/ad.py [hunk 2]
```diff
@@ -1,3 +1,7 @@
+"""Taichi automatic differentiation module.
```
Comment (k-ye) [inline python/taichi/ad.py:1]: Here and below: there's a new line after the first short description. See https://github.com/google/styleguide/blob/gh-pages/pyguide.md#382-modules

File: python/taichi/aot/__init__.py [hunk 3]
```diff
@@ -1,2 +1,6 @@
+"""Taichi's AOT (ahead of time) module. Users can use Taichi
```
Comment (k-ye) [inline python/taichi/aot/__init__.py:1]:
```suggestion
"""Taichi's AOT (ahead of time) module. 

Users can use Taichi ...
```

File: python/taichi/aot/record.py [hunk 4]
```diff
@@ -18,10 +18,39 @@ def record_action_config(key, value):
 
 
 def start_recording(filename):
+    """Start recording kernel information to a `yml` file.
```
Comment (k-ye) [inline python/taichi/aot/record.py:21]: Here and below: All the verbs should be in third person.

```suggestion
    """Starts recording kernel information to a `yml` file.
```

**Review by k-ye — APPROVED** [review]
I got a few nits, but thanks for filling all the docstrings, LGTM!

File: python/taichi/lang/impl.py (hunk 1 above)
Comment (neozhaoliang) [reply python/taichi/lang/impl.py]: Sure. fixed.

**Review by k-ye — APPROVED** [review]
lgtm!

**Pull request merged**